xdpderp/headers/LICENSE.BSD-2-Clause

```diff
@@ -0,0 +1,32 @@
+Valid-License-Identifier: BSD-2-Clause
+SPDX-URL: https://spdx.org/licenses/BSD-2-Clause.html
+Usage-Guide:
+  To use the BSD 2-clause "Simplified" License put the following SPDX
+  tag/value pair into a comment according to the placement guidelines in
+  the licensing rules documentation:
+    SPDX-License-Identifier: BSD-2-Clause
+License-Text:
+
+Copyright (c) <year> <owner> . All rights reserved.
```

Comment:
This would need to be filled in by libbpf maintainers. They did fix this, and we needed to bump the deps anyway past their fix, so resolved in https://github.com/tailscale/tailscale/pull/12390/commits/729de14769435f0587b47a5b7b12fe4a70dd8ba9